eahub/templates/eahub/edit_profile.html

```diff
@@ -1,81 +1,224 @@
+            </div>
+          </div>
+          <div class="row row-cols-2">
+            <div class="col">
+              {% include 'includes/field.html' with form=form field=form.country %}
+            </div>
+            <div class="col">
+              {% include 'includes/field.html' with form=form field=form.city_or_town %}
+            </div>
+          </div>
+          {% include 'includes/field.html' with form=form field=form.image %}
+
+          {% include 'includes/field.html' with form=form rows=7 field=form.summary placeholder="In West Philadelphia born and raised. On the playground is where I spent most of my days." %}
+
+          <div class="mb-5"
+            data-type-name="generic"
+            data-type-label="Tags"
+          >
+            <div class="vue-profile-tag-input"></div>
+          </div>
+
+          <div class="row row-cols-1 row-cols-lg-3">
+            <div class="col">
+              {% include 'includes/field.html' with form=form field=form.facebook_url %}
+            </div>
+            <div class="col">
+              {% include 'includes/field.html' with form=form field=form.linkedin_url %}
+            </div>
+            <div class="col">
+              {% include 'includes/field.html' with form=form field=form.personal_website_url placeholder="https://example.com" %}
+            </div>
+          </div>
+
+          <div class="row row-cols-1 row-cols-lg-2">
+            <div class="col">
+              {% include 'includes/field.html' with form=form rows=2 field=form.offering placeholder="Teaching python" %}
+            </div>
+            <div class="col">
+              {% include 'includes/field.html' with form=form rows=2 field=form.looking_for placeholder="Chat about wild animal suffering" %}
+            </div>
+          </div>
+
+
+          <h2>Community</h2>
+          {% include 'includes/field.html' with form=form field=form.available_as_speaker %}
+
+          {% if profile.topics_i_speak_about %}
+            {% include 'includes/field.html' with form=form rows=1 field=form.topics_i_speak_about %}
+          {% endif %}
+          <div class="mb-5"
+            data-type-name="speech_topic"
+            data-type-label="Topics I speak about"
+          >
+            <div class="vue-profile-tag-input"></div>
+          </div>
+
+          {% include 'includes/field.html' with form=form field=form.available_to_volunteer %}
+
+          {% include 'includes/field.html' with form=form field=form.local_groups %}
+
+          <div class="mb-5"
+            data-type-name="organisational_affiliation"
+            data-type-label="Organisational affiliations"
+          >
+            <div class="vue-profile-tag-input"></div>
+          </div>
+          
+          <div class="mb-5"
+            data-type-name="pledge"
+            data-type-label="Pledges"
+          >
+            <div class="vue-profile-tag-input"></div>
+          </div>
+
+          <div class="mb-5"
```

Comment:
I only get this locally, not on the deployed server: On selecting an event, this occurs:
![image](https://user-images.githubusercontent.com/15908208/112799331-1c40ba00-9066-11eb-9e55-1a5a1a91fd7d.png)

Any idea how this could be fixed? Not urgent prior to release, but ideally we have the same functionality locally

Reply:
Did you clone the database from the tags branch before? This list comes from algolia index from the tags db, it won't work without the identical SQL rows.